Source/common/SNTBlockMessage.m

```diff
@@ -177,6 +181,10 @@ + (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event customURL:(NSString *)
       ^{ return event.fileSHA256 ? event.fileBundleHash ?: event.fileSHA256 : nil; },
```

Comment:
Can we use this opportunity to change this dictionary to be initialized with literal syntax so we don't have to read the keys/values backwards?

Reply:
Done, simplified a bit too. I don't know why the values were blocks? I have replaced them with values.

@mlw Any thoughts on this?

Reply:
IIRC they were blocks so that we didn't call all of the properties and store their values in the dictionary when most of the values would not be used.

Reply:
Oh, to handle the event.xxx nil case.

Reply:
> IIRC they were blocks so that we didn't call all of the properties and store their values in the dictionary when most of the values would not be used.

All the blocks are executed, I don't think its to save reads from the event. I think it's just a way to get around having a nil value in a dict.

Reply:
Yeah don't think its worth it, this isn't a hot code path. I just went with NSNull.

Reply:
As mentioned above, my recollection was just about not storing nil in the dictionary. The change to always store NSNull and then check for that value is fine too.